doc/nrf/ug_bootloader.rst

```diff
@@ -77,12 +78,23 @@ It is protected by the root of trust in form of the immutable bootloader, and it
 The upgradable bootloader should carry out the same signature and metadata verification as the immutable bootloader.
 In addition, it can provide functionality for upgrading both itself and the following image in the boot sequence (in most cases, the application).
 
-A default implementation of an upgradable bootloader is not available yet.
+To ensure that there is a fallback solution in case of a faulty upgrade, the upgradable bootloader is stored in one of two possible partitions (slot 0 and slot 1).
```

Comment:
Should we provide motivation here? If so, the biggest reason for using this scheme is to have a simpler first stage bootloader, as it does not require flash write support when we use the two-slot solution.